_layouts/software-details.html

```diff
@@ -73,8 +73,12 @@ <h3 class="color-content font-serif my-2 my-md-4">
             <div class="col-sm">
               <p><span class="label">{{ t.software.technical_contact }} </span>
                 {% for contact in page.publiccode.maintenance.contacts limit: 1 %}
-                <a href="mailto:{{ contact.email }}">{{ contact.name }}</a>
-                {{ contact.phone }}
+                    {% if contact.email != nil %}
```

Comment:
Did you test this? You might also need to test if `contact.email != ""` because Jekyll is annoying.

Reply:
Thanks! I followed the path of the other checks on strings like `name` but I agree. Should we propagate the double check (null/empty) on all string vars?